ArticleClass.js

```diff
@@ -0,0 +1,32 @@
+export class ARTICLE {
```

Comment:
JavaScript에서 클래스명은 일반적으로 PascalCase(첫 글자만 대문자)로 작성합니다. 
ARTICLE 대신 Article로 작성하면 다른 개발자들이 보기에도 더 자연스럽습니다. 
대문자로만 쓰는 것은 보통 상수에 사용됩니다.
```suggestion
export class Article {
```